plugin-flex-ts-template-v2/src/feature-library/enhanced-crm-container/custom-components/IFrameCRMContainer/IFrameWrapper/IFrameWrapper.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { Component } from 'react';
+import * as Flex from '@twilio/flex-ui';
+
+import { wrapperStyle, frameStyle } from './IFrameWrapperStyles'
+
+const IFrameRefreshButtonStyledDiv = Flex.styled('div')`

```

Comment:
This could go in IFrameWrapperStyles.

If you change the IconButton to use `variant='primary'` then you can remove a lot of these styles, only these would remain:

```
const IFrameRefreshButtonStyledDiv = Flex.styled('div')`
  border-radius: 50%;
  bottom: 10px;
  box-shadow: rgb(0 0 0 / 30%) 0px 1px 10px;
  cursor: pointer;
  position: fixed;
  right: 10px;
  z-index: 9999;
`;
```